java/registry/src/main/java/dev/sunbirdrc/registry/controller/RegistryEntityController.java

```diff
@@ -110,17 +116,36 @@ public ResponseEntity<Object> invite(
         }
     }
 
+    @NotNull
+    private static ResponseParams createSchemaNotFoundResponse(String entityName, ResponseParams responseParams) {
+        ResponseParams responseParams1;
+        String errorMessage = String.format(ErrorMessages.NOT_PART_OF_THE_SYSTEM_EXCEPTION, entityName);
```

Comment:
we can get the error message passed into the function of createSchemaNotFoundResponse. we are already throwing the error with appropriate message in checkEntityNameInDefinitionManager function

Reply:
Not getting exactly what you mean to say